nextcord/enums.py

```diff
@@ -2037,6 +2039,30 @@ class ForumLayoutType(IntEnum):
     """Display posts as a collection of posts with images, this is more image focused."""
 
 
+class OnboardingPromptType(IntEnum):
+    """Represents the type of :class:`OnboardingPrompt`.
+
+    .. versionadded:: 3.0
+    """
+
+    multiple_choice = 0
+    """Displays the options as multiple choice."""
+    dropdown = 1
+    """Displays the options in a dropdown menu."""
+
+
+class OnboardingMode(IntEnum):
+    """The criteria needed in order for onboarding to be enabled.
+
+    .. versionadded:: 3.0
+    """
+
+    default = 0
+    """Only Default Channels are needed."""
```

Comment:
Might make sense to `"` quote "Default Channels" and similar phrases used in the UI.